docs/analyzer/user_guide.md

```diff
@@ -1393,6 +1393,16 @@ positional arguments:
 
 optional arguments:
   -h, --help            show this help message and exit
+  --config CONFIG_FILE  Allow the configuration from an explicit JSON based
```

Comment:
I checked the user guide and at the configuration file section (https://github.com/Ericsson/codechecker/blob/master/docs/analyzer/user_guide.md#configuration-file-) in the Example the codechecker.json file link is not working. Could you please fix this too in this pr? It should point to this file: https://github.com/Ericsson/codechecker/blob/master/config/codechecker.json.

It would be also great to extend this file with more examples for store, server and parse subcommands too. Unfortunately we forgot to extend this file :/